contrib/pthreadw32/module.defs

```diff
@@ -12,13 +12,13 @@ PTHREADW32.BUILD.ntargets = clean GC-static
 
 define PTHREADW32.INSTALL
     $(CP.exe) $(PTHREADW32.EXTRACT.dir/)pthread.h $(CONTRIB.build/)include/
-    $(SED.exe) -i.sedbak -e 's/ __declspec (dllexport)//g' -e 's/ __declspec (dllimport)//g' $(CONTRIB.build/)include/pthread.h
+    $(SED.exe) -i.sedbak -e 's/ __declspec (dllexport)//g' -e 's/ __declspec (dllimport)//g'  -e 's/__declspec (dllimport) //g' $(CONTRIB.build/)include/pthread.h
```

Comment:
Actually I suppose for consistency I should also check for the version without space for dllexport too?